win_configure/msi/wix/prep.bat

```diff
@@ -88,9 +88,10 @@ if %ERRORLEVEL% == 0 (
 		"%binaries_path%\tcltk\bin\*.dll"
 	) do (
 	xcopy /s /d "%%~a" "%EXECDIR%\bin" > NUL
+	if NOT %ERRORLEVEL% == 0 goto theend
 	)
 ) else (
-echo Fail to create %EXECDIR%\bin directory.
+echo Failed to create %EXECDIR%\bin directory.
 goto theend
```

Comment:
Ident this lines properly

Reply:
Thanks @subhasisb  for your review comments, 

I have pushed new commit with changes, please have a look.